cpanfile

```diff
@@ -1,6 +1,6 @@
 ##################################################
 # WARNING
-# This file is autogenerated by tools/update-deps
+# This file is autogenerated by ../../../../../../../home/iob/os-autoinst-distri-opensuse/os-autoinst/tools/update-deps
```

Comment:
Huh?

Reply:
awesome

Reply:
should i fix that manually?

Reply:
I guess that's what make the test unhappy.

Reply:
> should i fix that manually?

I can't reproduce that when I run `make update-deps` locally.
Did you run this from the os-autoinst root folder?

Reply:
yes

Reply:
the `$proj_root` is set to `/var/lib/openqa/share/tests/os-autoinst-distri-opensuse/os-autoinst/tools/..Updated dist/rpm/os-autoinst.spec`

Reply:
```❯ ll ~/os-autoinst-distri-opensuse
lrwxrwxrwx 1 iob users 55 Jun  4  2020 /home/iob/os-autoinst-distri-opensuse -> /var/lib/openqa/share/tests/os-autoinst-distri-opensuse```

Reply:
ok, I guess you need to fix this manually for now.
Since you are maybe the only one who can reproduce it:
Could you add the following lines in tools/update-deps line 30 for debugging and paste the output?
```
my $FILE = __FILE__;
my $filepath = path(__FILE__);
say "proj_root=$proj_root __FILE__=$FILE filepath=$filepath";
```
and also which perl version you are running

Reply:
perl 5, version 38, subversion 2 (v5.38.2)

Reply:
output: 
```❯ make update-deps
ninja -C /var/lib/openqa/share/tests/os-autoinst-distri-opensuse/os-autoinst/build update-deps
ninja: Entering directory `/var/lib/openqa/share/tests/os-autoinst-distri-opensuse/os-autoinst/build'
[1/1] Updating dependency information
proj_root=/var/lib/openqa/share/tests/os-autoinst-distri-opensuse/os-autoinst/tools/.. __FILE__=/home/iob/os-autoinst-distri-opensuse/os-autoinst/tools/update-deps filepath=/home/iob/os-autoinst-distri-opensuse/os-autoinst/tools/update-deps
Updated dist/rpm/os-autoinst.spec
Updated /var/lib/openqa/share/tests/os-autoinst-distri-opensuse/os-autoinst/tools/../cpanfile
Updated container/os-autoinst_dev/Dockerfile
```

Reply:
i guess my case can be explained by Oli with rocket science :) idk